components/pkg-export-helm/src/chart.rs

```diff
@@ -139,10 +139,11 @@ impl<'a> Chart<'a> {
             Status::Creating,
             format!("templates directory `{}`", template_path),
         )?;
-        fs::create_dir_all(&template_path)?;
-        self.generate_manifest_template(&template_path)?;
 
-        self.generate_values()
+        self.generate_values()?;
```

Comment:
This change is strange - there is nothing in this commit that changes the `generate_values` function. Is it fixing some bug?